qiime/sdk/visualization.py

```diff
@@ -33,3 +35,7 @@ def _from_data_dir(cls, data_dir, provenance):
         # shutil.copytree doesn't allow the destination directory to exist.
         viz._archiver.save_data(data_dir, distutils.dir_util.copy_tree)
         return viz
+
+    def get_indices(self):
```

Comment:
I'm rethinking this API... Should this be a class method instead, and return tuples of extension and path (e.g., ``("tsv", "data/index.tsv")``)? That's more in line with how I'd use it for the cli. I'm open to suggestions here - I don't really like this as is.